### DISCUSSION TITLE
can we have multiple values in a key for pinecone metadata filtering ex {"Date" :["24-09-2024","25-09-2024"}.

### DISCUSSION THREAD

**Discussion by hasanali28** [FlowiseAI/Flowise#3142] [Q&A]
![Screenshot 2024-09-04 at 10 16 00 AM](https://github.com/user-attachments/assets/70e02321-10db-4155-8e4b-ab444d5e3b37)
We have the functionality in Pinecone to send multiple values for the same key but wondering if we can somehow do the same in Flowise. Need desperate help here.

**Comment by amansoni7477030** [comment]
Hii @hasanali28 its possible you can use pinecone metadata filter in flowise
